src/com/lftechnology/task1/jan12/LinkedListStudy.java

```diff
@@ -0,0 +1,41 @@
+package com.lftechnology.task1.jan12;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <h1>Q3.</h1>
+ * <ul>
+ * <li>Create a linked list named one and two.</li>
+ * <li>Add some elements to both one (one, two, three, four, five) and two (six, seven, eight, nine, and ten).</li>
+ * <li>Merge the words from two into one.</li>
+ * <li>Remove every second word from two.</li>
+ * <li>Remove all words in two from one.</li>
+ * <li>Print the value after each operation.</li>
+ * </ul>
+ * 
+ * @author Sanjib<sanjibmaharjan@lftechnology.com>
+ * 
+ */
+public class LinkedListStudy {
+	public static final Logger LOGGER = Logger.getLogger(LinkedListStudy.class.getName());
+
+	public static void main(String[] args) {
+		List<String> linkedListOne = new LinkedList<String>(Arrays.asList(new String[] { "one", "two", "three", "four", "five" }));
+		List<String> linkedListTwo = new LinkedList<String>(Arrays.asList(new String[] { "six", "seven", "eight", "nine", "ten" }));
+		LOGGER.log(Level.INFO, "Original values in the lists\nLinked list1:{0}\nLinked list2:{1} ",
```

Comment:
You could have simply printed the linkedListOne instead of calling the Utils.listToString() method